integrations/extensions/starter-kits/watson-discovery/README.md

```diff
@@ -0,0 +1,114 @@
+2. We show how to restrict the search on a specific topic to a single collection that is part of a larger project in Watson Discovery v2.  (In doing so, we show how to set any of the Watson Discovery parameters to any value you want, which could also be applicable to many other needs.)
+
+Here are some additional examples of things you might need to do when connecting Watson Assistant to Watson Discovery that could require the added flexibility of a custom extension:
+
+1. If your Watson Discovery documents have a complex schema with many fields (which can happen if you are uploading CSV files or JSON files or if you have a customized Smart Document Understanding model), the custom extension approach allows you to use any or all of the fields you want.  (In contrast, the built-in integration only lets you select one title field, one text field, and one URL field).
+2. If your Watson Discovery documents include fields that are arrays or complex nested structures, you can concatenate or otherwise arrange elements from those arrays or structures.  (In contrast, the built-in integration will take the first element of each array only and will only allow you to pick a single atomic field within a nested structure).
+3. You can use metadata in a field in your schema to set the color or shape or other visual elements of a search response.
+4. You can show a small graph or table in your response card (or even have a larger one pop up when you click or mouse over).
+5. You can get information from custom enrichments in Watson Discovery (e.g., entities or text classifications) and use that information to affect how results are presented to the user.
+
+More broadly, anything you can do with Watson Discovery parameters and Watson Discovery outputs, you can do with a custom extension (at least within the size limitation: see the "Limit on Size of Search Results" section later in this document for details).  However, a custom extension will be more work, so if you just need simple search results, consider using with the [built-in search integration](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-search-add).
+
+### Setting up Watson Discovery
+
+Before starting, you will need an instance of IBM Watson Discovery, and you will need the API key and instance ID for that instance.  You will also need to know the region where the instance is located, which is found in the URL for the instance immediately after `https://api.` For example if the URL for your instance is `https://api.us-south.discovery.watson.cloud.ibm.com/instances/447fc929-fde9-402e-b73c-03274b8ab0bb`, then the region where your instance is located is `us-south` and your instance ID is `447fc929-fde9-402e-b73c-03274b8ab0bb`.  The URL and API key are both found in the "Credentials" section of the resource page for the instance in IBM Cloud.
```

Comment:
Should we mention that discovery would need a separate paid plan?

Reply:
Yeah that makes sense.

Reply:
Done!